traits/src/memory.rs

```diff
@@ -43,6 +39,11 @@ impl IndexMut<Addr> for MemoryStub {
 }
 
 impl Memory for MemoryStub {
+
+    fn read_word(&self, addr: Addr) -> Word{0 as u16}
+
+    fn write_word(&mut self, addr: Addr, word: Word){}
+    
```

Comment:
This is technically a change in behavior from the defaulted `read_word`/`write_word` on `MemoryStub` which would return your last write but that's fine; I don't think we were relying on this anywhere.